opae.spec

```diff
@@ -26,11 +27,13 @@ BuildRequires:  doxygen
 BuildRequires:  systemd
 BuildRequires:  pybind11-devel
 BuildRequires:  python3-setuptools
+BuildRequires:  spdlog-devel
 BuildRequires:  tbb-devel
 BuildRequires:  git
 BuildRequires:  python3-jsonschema
 BuildRequires:  python3-pip
 BuildRequires:  python3-virtualenv
```

Comment:
These can be removed (maybe in a later PR)

Reply:
yes, a later pr.  this is just fedora building, rhel is next